cmd/remotefs/azurefs.go

```diff
@@ -324,7 +401,37 @@ func containerMountAzureFilesystem(tempDir string, index int, fs AzureFilesystem
 	}
 	logrus.Debugf("Device opened: %s", deviceName)
 
-	// 4) Mount block device as a read-only filesystem.
+	// 4) Config the decrypted filesystem in dm-verity format

```

Comment:
Surely some check as to whether this is required or not should happen here?

Reply:
Yes I think dm-verity should be an config option